src/Parser.js

```diff
@@ -94,6 +94,25 @@ export function parse (value, root, parent, rule, rules, rulesets, pseudo, point
 
 				index = offset = property = 0, variable = ampersand = 1, type = characters = '', length = pseudo
 				break
+			// \
+			case 92:
+				if (from(peek()).match(/[^a-f0-9]/i)) {
+					characters += '\\'
+					characters += from(next())
+				} else {
+					characters += '\\';
+					[1,2,3,4,5,6].some(() => {
```

Comment:
I hesitated to add a `for(var i` loop, as all the variables are hoisted and use the `var` keyword - I didn't want to pollute that list with a scarcely-used `i`.